cinema/views.py

```diff
@@ -21,24 +31,49 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin
+):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
+    authentication_classes = (TokenAuthentication,)
```

Comment:
maybe it would be better to set DEFAULT_AUTHENTICATION_CLASSES and DEFAULT_PERMISSION_CLASSES in settings.py instead of writing authentication_classes and permision_classes to each view. 
But if you'll do so, don't forget to rerun tests for user as well, couse you may like to change some permission_classes there.